### PR TITLE
Fix chart storage keyfile issue in gcs

### PR DESCRIPTION
**Pull request by ninjadq**
Add volumn binding on docker-compose.yml

Signed-off-by: Qian Deng <dengq@vmware.com>

### CODE REVIEW

File: make/photon/prepare/templates/docker_compose/docker-compose.yml.jinja [hunk 1]
```diff
@@ -378,6 +378,9 @@ services:
     volumes:
       - {{data_volume}}/chart_storage:/chart_storage:z
       - ./common/config/chartserver:/etc/chartserver:z
+{% if gcs_keyfile %}
```
Comment (stonezdj) [inline make/photon/prepare/templates/docker_compose/docker-compose.yml.jinja]: Please do not take gcs_keyfile as a condition, this file is generated with empty content in harbor tile env, use storage_service is gcp

Comment (reasonerjt) [inline make/photon/prepare/templates/docker_compose/docker-compose.yml.jinja]: This attribute is not reliable:
https://github.com/goharbor/harbor/blob/master/make/photon/prepare/utils/docker_compose.py#L37

You should make sure this attribute is set only when the storage is `"gcs"`

File: make/photon/prepare/utils/chart.py [hunk 2]
```diff
@@ -12,13 +12,12 @@
 def prepare_chartmuseum(config_dict):
 
     core_secret = config_dict['core_secret']
-    registry_custom_ca_bundle_path = config_dict['registry_custom_ca_bundle_path']
```
Comment (reasonerjt) [inline make/photon/prepare/utils/chart.py]: How is this change relevant to this PR?  Is it necessary to remove it?

Comment (ninjadq) [reply make/photon/prepare/utils/chart.py]: because it is an unused var
